### ISSUE TITLE
scale line control does not always show

### ISSUE THREAD

**Issue by bartvde** [openlayers/ol3#3444]
To reproduce go to the scale-line example, choose nautical miles and type map.getView().setZoom(15) into the console.

Initially reported here:

http://stackoverflow.com/questions/29333183/openlayer3-scaleline-does-not-show-nautical

What happens is that i becomes -3 in

```
  var i = 3 * Math.floor(
      Math.log(this.minWidth_ * pointResolution) / Math.log(10));
```

and the div will be hidden. I'm guessing the code needs to be enhanced to show fractional values in this case.

**Comment by aAXEe** [comment]
You can also see this behaviour at the official example: http://openlayers.org/en/v3.12.1/examples/scale-line.html
The nautical scale-line shows up down to a scale of 1nm. If you zoom in further it disappears.

**Comment by ahocevar** [comment]
Will be fixed with #4629.

**Comment by aAXEe** [comment]
Sounds great. Thank you!
